src/util/sss_semanage.c

```diff
@@ -229,6 +262,20 @@ static int sss_semanage_user_mod(semanage_handle_t *handle,
     return ret;
 }
 
+int get_seuser(const char *linuxuser,
```

Comment:
IMHO better name would be sss_getseusername. But I would prefer a different solution: to inline this function
and do not include "selinux/selinux.h" into this file.

Reply:
About the name change, did you mean get_seuserbyname (with 'by' in the middle)? I am OK with that name. But without the 'by' part it does not sound right, because it also returns the level, not just name.

I am not sure about the inlining. Inlining would be done in the header file, but it uses static function local to the .c file. So I think it is better to keep the function not inlined and in the .c file.

What do you think? I will update the patch based on you reply.